src/igvfd/tests/test_audit_construct_library.py

```diff
@@ -6,27 +6,8 @@ def test_audit_construct_library_associated_disease(
     base_construct_library,
     phenotype_term_alzheimers
```

Comment:
I think phenotype_term_alzheimers can be dropped from this as it's no longer used in the test code.

Reply:
Thanks for catching! Removed